frontend/src/pages/PipelineDetails.tsx

```diff
@@ -187,6 +187,14 @@ class PipelineDetails extends Page<{}, PipelineDetailsState> {
       versions,
     } = this.state;
 
+    const isSafari =
```

Comment:
Can you add a reference comment where the code comes from and explain why we need to add the hack?

Reply:
@Bobgy , addressed, please check!